package.json

```diff
@@ -43,7 +43,7 @@
     "react-codemirror": ">=0.1.2",
     "react-dom": ">=15.5.0",
     "rollup": "^0.62.0",
-    "rollup-plugin-babel": "^4.0.0-beta.7",
+    "rollup-plugin-babel": "^4.0.0-beta.2",
```

Comment:
what is the reason for these "downgrades" (+babel-loader)
They both resolved to beta.7 in yarn.lock, but still

Reply:
According to the official upgrade command "npx babel-upgrade",  the files will be modified automatically.So I didn't modify these configurations myself.

[Official documents](https://babeljs.io/blog/2018/07/27/removing-babels-stage-presets#migrating)

Reply:
It does not mean it is perfect and we do not need to check if all these automatic changes are actually necessary 🤷‍♂️

Reply:
I upgraded by hands in https://github.com/chenglou/react-motion/pull/548

Reply:
ok, thank you for reply, i learn a lot.